docs/conf.py

```diff
@@ -52,7 +52,7 @@
 master_doc = "index"
 
 # General information about the project.
-project = "Web3.py"
+project = "web3.py"
 copyright = "2018, Piper Merriam, Jason Carver"
```

Comment:
Should we update this too? At least the date, not sure about the names?

Reply:
+1... looks like we'd been moving towards "YYYY, The Ethereum Foundation" in all the other libraries.

Reply:
official foundation site copyright left off the "The", so i did too 🤝